code/Game.cs

```diff
@@ -5,134 +5,149 @@ namespace TTT;
+        player.BaseKarma = Karma.DefaultValue;
+        player.ActiveKarma = player.BaseKarma;
+
+        State.OnPlayerJoin( player );
+
+        UI.ChatBox.AddInfo( To.Everyone, $"{client.Name} has joined" );
+    }
+
+    public override void ClientDisconnect( Client client, NetworkDisconnectionReason reason )
+    {
+        Event.Run( GameEvent.Client.Disconnected, client, reason );
+        State.OnPlayerLeave( client.Pawn as Player );
+
+        UI.ChatBox.AddInfo( To.Everyone, $"{client.Name} has left ({reason})" );
+
+        // Only delete the pawn if they are alive.
+        // Keep the dead body otherwise on disconnect.
+        if ( client.Pawn.IsValid() && client.Pawn.IsAlive() )
+            client.Pawn.Delete();
+
+        client.Pawn = null;
+    }
+
+    public override void RenderHud()
+    {
+        base.RenderHud();
+
+        if ( Local.Pawn is not Player player )
+            return;
+
+        var scale = Screen.Height / 1080.0f;
+        var screenSize = Screen.Size / scale;
+        var matrix = Matrix.CreateScale( scale );
+
+        using ( Render.Draw2D.MatrixScope( matrix ) )
+            player.RenderHud( screenSize );
+    }
+
+    public override bool CanHearPlayerVoice( Client source, Client dest )
+    {
+        if ( !source.Pawn.IsAlive() && !dest.Pawn.IsAlive() )
+            return true;
+
+        if ( State is InProgress && !source.Pawn.IsAlive() && dest.Pawn.IsAlive() )
+            return false;
+
+        return true;
+    }
+
+    public override void OnVoicePlayed( Client client )
+    {
+        UI.VoiceChatDisplay.Instance?.OnVoicePlayed( client );
+    }
+
+    public override void PostLevelLoaded()
+    {
+        base.PostLevelLoaded();
+
+        ForceStateChange( new WaitingState() );
+    }
+
+    [Event.Tick]
+    private void Tick()
+    {
+        State?.OnTick();
+    }
+
+    private static void LoadResources()
+    {
+        Detective.Hat = ResourceLibrary.Get<Clothing>( "models/detective_hat/detective_hat.clothing" );
+
+        Player.ClothingPreset.Add( ResourceLibrary.Get<Clothing>( "models/citizen_clothes/hat/balaclava/balaclava.clothing" ) );
+        Player.ClothingPreset.Add( ResourceLibrary.Get<Clothing>( "models/citizen_clothes/hair/eyebrows/eyebrows_black.clothing" ) );
+        Player.ClothingPreset.Add( ResourceLibrary.Get<Clothing>( "models/citizen_clothes/jacket/longsleeve/longsleeve.clothing" ) );
+        Player.ClothingPreset.Add( ResourceLibrary.Get<Clothing>( "models/citizen_clothes/gloves/tactical_gloves/tactical_gloves.clothing" ) );
+        Player.ClothingPreset.Add( ResourceLibrary.Get<Clothing>( "models/citizen_clothes/trousers/smarttrousers/trousers.smart.clothing" ) );
+        Player.ClothingPreset.Add( ResourceLibrary.Get<Clothing>( "models/citizen_clothes/vest/tactical_vest/models/tactical_vest.clothing" ) );
+        Player.ClothingPreset.Add( ResourceLibrary.Get<Clothing>( "models/citizen_clothes/shoes/trainers/trainers.clothing" ) );
+    }
```

Comment:
Put this method in `Player.Clothes.cs` imo

Reply:
I think this is fine.